### ISSUE TITLE
Press F1 for Help on Accessibility screen in Setup doesn't work

### ISSUE THREAD

**Issue by dftf-stu** [fabiangreffrath/crispy-doom#1144]
### Background

Crispy Doom: 6.0.0 (31 Mar 2023)
Operating System: Windows 10 Home (x64) 22H2 Build 19045.3930
Game: DOOM / DOOMII / Master Levels / Plutonia / TNT / Freedoom Phase 1 / Freedoom Phase 2 (etc.)

### Bug description

**Observed behavior:**

I wanted to see if there was an option in the "Accessibility" section to make levels be constantly fully-lit ("full-bright", I think is the DOOM term -- though the GameBoy Advance ports referred to this as "static lighting"), as on some devices (especially those with OLED screens) I am finding some darker-areas almost-impossible to see.

My only current workaround is to type **IDBEHOLDL**, but this only last for two-minutes, so this isn't ideal.

I went into Crispy DOOM Setup, then to Accessibility, but I'm not-sure what all of the options do.

![CDA1](https://github.com/fabiangreffrath/crispy-doom/assets/83965343/ea98a1cc-45b3-4a12-967e-bd2d76513eb3)

I pressed F1 to go to a help-page, but it takes me to a page that doesn't exist:

![CDA2](https://github.com/fabiangreffrath/crispy-doom/assets/83965343/ea736280-b2d9-49cc-8f78-5fb428e5d2ba)

**Expected behavior:**

That when pressing F1, it would take me to a page that explains what each option does.

To have an accessibility-option that would do as I suggested, so I don't have to repeatedly type **IDBEHOLDL** every two-minutes

**Comment by kitchen-ace** [comment]
Definitely a bug to not have those help screens up. In the meantime turn up Extra Lighting to 8, to have (nearly) full brightness.

**Comment by fabiangreffrath** [comment]
We rely on Chocolate Doom's online help for the different pages in the setup tool, but Chocolate Setup of course doesn't have an Accessibility page.

**Comment by dftf-stu** [comment]
@fabiangreffrath: could you not perhaps host the page for that on here?

For example: create a ticket on here which explains the options, then close it, and then link to that page in Crispy?  Or are you saying the URLs are not something you can change (i.e. they are hard-coded into the bits of code you take from Chocolate)?
